src/_guides/json.md

```diff
@@ -0,0 +1,55 @@
+---
+title: "JSON Support"
+description: "Dart solutions for reading and writing JSON."
+---
+
+Most mobile and web apps use JSON for tasks such as
+exchanging data with a web server.
+This page discusses Dart support for JSON _serialization_ and _deserialization_:
+converting Dart objects _to_ and _from_ JSON.
+
+
+## Generally useful libraries
+
+The following libraries and packages are useful across Dart platforms:
+
+* [dart:convert](/guides/libraries/library-tour#dartconvert---decoding-and-encoding-json-utf-8-and-more)<br>
+  Converters for both JSON and UTF-8
+  (the character encoding that JSON requires).
+
+* [package:json_serializable](https://pub.dartlang.org/packages/json_serializable)<br>
+  Easy-to-use support for using the Dart build system to generate code that
+  serializes and deserializes Dart objects.
+
+* [package:built_value](https://pub.dartlang.org/packages/built_value)<br>
+  A powerful, opinionated alternative to json_serializable.
+
+
+## Flutter resources
+
+[JSON and serialization](https://flutter.io/json/)
+: Shows how Flutter apps can serialize and deserialize both
+manually (with dart:convert) and automatically (with json_serializable).
```

Comment:
I'd suggest dropping the "manually" and "automatically" qualifiers. Doing so will simplify the description and avoid possible confusion about the meaning of those qualifiers.